packages/preview/sunny-famnit/0.2.0/examples/example.typ

```diff
@@ -0,0 +1,85 @@
+#import "@preview/sunny-famnit:0.2.0": project
+#import "attachments.typ" as pr
+
+#let input = (
+  izvleček: [
+    To je krako delo spisano za pomoč pri uporabi Typsta za delo zaključnega dela. V izvlečku se uporabi največ 250 besed. Tu se izpiše probleme, metode, rezultate, ključne ugotovitve in njihov pomen.
+  ],
+  abstract: [
+    This is a stub written to help you with use of Typst for thesis work. A maximum of 250 words is used in the abstract. Problems, methods, results, key findings and their significance are listed here.
+  ],
+  ključne_besede: ("Typst", "Zaključna naloga", "Pisanje", "Navodila"),
+  key_words: ("Typst", "Final work", "Writing", "Tutorial"),
+  kratice: (
+    "Famnit": "Fakulteta za matematiko naravoslovje in informacijske tehnologije",
+    "PDF": "Portable document format",
+  ),
+  priloge: (
+    "Raw koda": raw(read("example.typ"), lang: "typst", block: true),
+    "some obscurly long ass text that would be so long that it goes into another line to test. But hey you still need more words for it to go there.": pr.a, // sample of importing priloge.
+  ),
+  zahvala: [
+    Opcijsko se lahko tu zahvalite typst-u za olajšanje dela. Če zahvale ne želite napisati, zamenjajte content( to so vsebinski bloki znotraj `[]`) z `none`
+  ],
+)
+
+#show: project.with(
+  date: datetime(day: 1, month: 1, year: 2024),
+  author: "Tilen Gimpelj",
+  studij: "Bioinformatika",
+  mentor:(
+    name: "Oge So-Kul", 
+    sl: ("dr.",""),
+    en: ("",".phD"),
+    ),
+  somentor: none, // optional somentor, written in the same way as mentor
+  work_mentor: none, // optional work mentor can be written the same way as mentor
```

Comment:
Prefer snake case for variable names

Reply:
I think you mean kebab-case? If yes, indeed, it's preferred. It's not a hard requirement, but I'll hold off on merging until there's a response.

Reply:
Whoops absolutely I meant kebab case (I dread to think how many times I have mixed those two up 😭)